fedot/preprocessing/preprocessing.py

```diff
@@ -542,3 +558,29 @@ def update_indices_for_time_series(self, test_data: Union[InputData, MultiModalD
                 last_id = len(input_data.idx)
                 input_data.idx = np.arange(last_id, last_id + input_data.task.task_params.forecast_length)
         return test_data
+
+    @copy_doc(BasePreprocessor.reduce_memory_size)
+    def reduce_memory_size(self, data: InputData) -> InputData:
+        if isinstance(data, InputData):
+            if data.task.task_type == TaskTypesEnum.ts_forecasting:
```

Comment:
At this point the value of the `data.supplementary_data.col_type_ids` field is set to `None`, resulting in integration tests failures for `test_classification.py:test_image_classification_quality`, `test_classification.py:test_cnn_methods`, and `test_data.py:test_data_from_image`